doc/nrf/app_dev/device_guides/thingy91x/thingy91x_recover_to_factory_firmware.rst

```diff
@@ -0,0 +1,64 @@
+.. _thingy91x_recover_to_factory_firmware:
+
+Recovering the Thingy:91 X to factory firmware
+##############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can recover the Thingy:91 X to factory firmware using the ``nrfutil device`` command.
+
+If your Thingy:91 X is powered on and connected using a USB cable, it shows up as a USB device.
+Else, something is preventing the :ref:`connectivity_bridge` application on the nRF5340 from running.
+For example, this could be an application running on the nRF9151 that is driving the wrong pins.
+Another typical case is that the nRF5340 has been flashed by accident.
+
+You need to use an external J-Link to follow these steps.
+You can also use a Nordic development kit with a debug-out port.
+
+See `Installing nRF Util`_ for instructions for installing nRF Util.
+Install the nRF Util device command as well.
```

Comment:
This is already covered by the Prerequisites for the entire SDK: https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/installation/install_ncs.html#install_prerequisites

Reply:
Removed the entries.